Redshift/BigQuery Linked Audiences GA + Materialized view requirements [DOC-1094] (#7554)
segmentio/segment-docs: src/unify/data-graph/index.md

## Prerequisites

> info "Why you need both materialized and unmaterialized tables"

> > Segment recommends using materialized views for Profiles Sync to optimize performance and reduce query costs with Linked Audiences. However, due to schema inference requirements, you'll still need to select the matching **unmaterialized tables**, too. Segment relies on them during setup, even if they’re not used when queries actually run.

Reviewer: ```suggestion
> Segment recommends using materialized views for Profiles Sync to optimize performance and reduce query costs with Linked Audiences. However, due to schema inference requirements, you still need to select the matching **unmaterialized tables** as well. Segment relies on the unmaterialized tables during setup, even if they’re not used when queries run.
```

To use the Data Graph, you'll need the following:

- A supported data warehouse with the appropriate Data Graph permissions
- Workspace Owner or Unify Read-only/Admin and Entities Admin permissions
- For Linked Audiences, set up [Profiles Sync](/docs/unify/profiles-sync/) in a Unify space with ready-to-use [data models and tables](/docs/unify/profiles-sync/tables/) in your warehouse. When setting up selective sync, Segment recommends the following settings: 
  - Under **Profile materialized tables**, select all the tables (`user_identifier`, `user_traits`, `profile_merges`) for faster and more cost-efficient Linked Audiences computations in your data warehouse.

>   - **Make sure to include the unmaterialized tables, too**. Segment needs them during setup to understand your schema.

Reviewer: ```suggestion
  - **Make sure to include the unmaterialized tables as well**. Segment needs them during setup to understand your schema.
```

Reviewer: should unmaterialized tables be bolded? Is it bolded in the UI?

Author: @stayseesong No. I'm trying to make the entire sentence bold, not just `unmaterialized` specifically. This is pretty key and we want to make sure users know they need both; apparently that message hasn't been super clear.
